pulsar-websocket/src/main/java/org/apache/pulsar/websocket/ConsumerHandler.java

```diff
@@ -84,6 +88,8 @@
     private volatile long msgDeliveredCounter = 0;
     private static final AtomicLongFieldUpdater<ConsumerHandler> MSG_DELIVERED_COUNTER_UPDATER =
             AtomicLongFieldUpdater.newUpdater(ConsumerHandler.class, "msgDeliveredCounter");
+    //default interval for checking if end of topic has been reached is 10 min
+    private static long END_OF_TOPIC_CHECK_INTERVAL = 10 * 60;
```

Comment:
Seems this one not used?

Reply:
my mistake, was trying to use push mode to notify endOfTopic with periodic check, probably can add that later.
removed it for now.